apps/api/src/app/inbox/utils/types.ts

```diff
@@ -38,3 +38,18 @@ export type NotificationFilter = {
   read?: boolean;
   archived?: boolean;
 };
+
+export type Workflow = {
+  id: string;
+  identifier: string;
+  name: string;
+  critical: boolean;
+  tags?: string[];
```

Comment:
I think we should update the Workflow type in `js/src/types.ts` file too to add the tags

Reply:
thanks, will do! 👍